src/TestFramework/TestFrameworkTypes.php

```diff
@@ -45,10 +51,40 @@ final class TestFrameworkTypes
     public const PHPSPEC = 'phpspec';
     public const CODECEPTION = 'codeception';
 
-    public const TYPES = [
+    /**
+     * @var string[]
+     */
+    private static array $defaultTypes = [
         self::PEST,
         self::PHPUNIT,
         self::PHPSPEC,
         self::CODECEPTION,
     ];
+
+    /**
+     * @param mixed[] $installedExtensions
+     *
+     * @return string[]
+     */
+    public static function getTypes(
+        array $installedExtensions = GeneratedExtensionsConfig::EXTENSIONS
+    ): array {
+        $types = self::$defaultTypes;
+
+        if (count($installedExtensions) > 0) {
```

Comment:
I see there's a warning from "[GitHub Actions / Mutation Testing Code Review Annotations 8.1](https://github.com/infection/infection/actions/runs/5350969967/jobs/9704233444?pr=1871)" on this line that changing `>` to `>=` is an escaped Mutant. I just confirmed that this case would be caught by `phpstan analyse`:

```
 ------ ---------------------------------------------------------------------
  Line   TestFramework/TestFrameworkTypes.php
 ------ ---------------------------------------------------------------------
  74     Comparison operation ">=" between int<0, max> and 0 is always true.
 ------ ---------------------------------------------------------------------
```